test/tools/isoltest.cpp

```diff
@@ -262,10 +299,6 @@ SyntaxTestStats SyntaxTestTool::processPath(
 				paths.pop();
 				++successCount;
 				break;
-			default:
```

Comment:
Do we have a compiler flag which will result in an error if we extend the `enum` but not this `switch`?

Reply:
This is usually the case at least on my machine.

Reply:
In this particular case even it shouldn't be a problem even if there is no compiler error - the only reason for extending this particular enum would be to handle distinguish cases in exactly this switch.